packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/rest/bar/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as Parameters from "./parameters.js";
+import * as Responses from "./responses.js";
+import * as Client from "./clientDefinitions.js";
+import * as OutputModels from "./outputModels.js";
+
+export { createClient } from "./barClient.js";
+export { Parameters, Responses, Client, OutputModels };
```

Comment:
In this case, we don't have isUnexpected generated, but in the bar subclient, and I wonder if we should use code like `if(isUnexpected)` in the modular level when calling the rest level ?

Reply:
Yeah, though we should see why isUnexpected was not generated?